_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: The ideal setup for a multi-user conda installation
+tags: conda bash systems-administration
+contributors:
+- boulund
+- pmenzel
+- hexylena
+---
+
+[Conda](https://docs.conda.io/projects/conda/en/stable/) and [Bioconda](https://bioconda.github.io/) have become crucial tools in most bioinformatician's toolkit. It is often used to install applications and create reproducible environments on their system or a compute cluster. 
+
+On multi-user systems (e.g. shared Linux servers), multiple users can share  the same conda installation and the environments thererin. This is a good idea and usually works well when only already existing environments are re-used, but can become tricky when users also want to create environments (using `conda create`) into the shared conda installation. On a standard Linux system, this often results in permission problems when (re-)downloading packages and overwriting files that were created by others.
+
+Here we describe a setup with a shared (read-only) conda installation, with re-usable shared environments, and private per-user environments.
+
+
+## Tutorial
+
+### Creating the shared conda installation
+
+
+First, create a shared conda installation folder that is read-only for all regular users. Shared conda environments therein, which are available to all users, must be created by the sysadmin (or with a user account with the correct privileges for the conda folder). 
+
```

Comment:
How would this folder be located/created when thinking of the main "conda" folder?

Reply:
Not sure what you mean by the main conda folder in this context?

Reply:
So, when you install conda, you make a directory and install it there. This is the "main conda folder". If you have installed conda a few times before, you know this. When you specify this directory here, I later on realize that this is the "main conda folder". However, when I read this just linearly from top to bottom, I got the feeling I needed an extra directory in addition to the "main conda folder".

Reply:
Yeah that type of confusion seems unnecessary and not what we want to introduce. What would be your preferred way of phrasing this to reduce the risk of confusion?

Reply:
The issue is setting the permissions here properly, so we need to mkdir it explictily. 

PREFIX is quite advanced knowledge, though.

Reply:
I rephrased things a bit, hope it makes it more clear.